### DISCUSSION TITLE
Development dependency?

### DISCUSSION THREAD

**Discussion by stacy-rendall** [ZeeCoder/use-resize-observer#87] [General]
The documentation **Install** section indicates this should be installed as a "dev" dependency, why is that? 

As per https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file, devDependencies are those that are only needed for local development and testing, whereas (as far as I can tell) this package would be carried through and used in production....?

**Comment by ZeeCoder** [comment]
It depends on your setup.
If you have a single repo responsible for both front- and backend, then node dependencies would be regular deps, and front-end ones would be Dev deps, as they would be bundled, but not shipped as-is.

**Comment by ZeeCoder** [comment]
I caved on this point in the end :sweat_smile: 
https://github.com/ZeeCoder/use-resize-observer/tree/alpha#install
